examples/chip-tool/commands/pairing/OpenCommissioningWindowCommand.h

```diff
@@ -40,7 +40,7 @@ class OpenCommissioningWindowCommand : public CHIPCommand
 
     /////////// CHIPCommand Interface /////////
     CHIP_ERROR RunCommand() override;
-    chip::System::Clock::Timeout GetWaitDuration() const override { return chip::System::Clock::Seconds16(5); }
+    chip::System::Clock::Timeout GetWaitDuration() const override { return chip::System::Clock::Seconds16(20); }
```

Comment:
Why does this fix it? Why would 15 not be enough? Why is 20 enough?

Unless the root cause of the timing is explained by a comment here, it would just be anecdotal and could still fail for some if 5 was not long enough, and neither was 15.

Reply:
I think we should do a few things:

1. Rename the current mTimeout member to mCommissioningWindowTimeout or something like that.
2. Add an optional mTimeout that would allow the consumer to control what `GetWaitDuration` returns, like other commands do.
3. Bump the default value.  Where to bump it to...  On an m5stack, CASE takes ~5 seconds to establish, so this current 5-second timeout is really pushing the envelope.  Past that we need to establish CASE, send a read, then send an open-commissioning-window.  Plus do some PBKDF work.  I'd think 10 seconds should work for that if CASE takes 5 seconds...  But really we just want it "long enough to not fail by accident too much", plus the explicit caller control to handle edge cases.